install/kubernetes/cilium/values.yaml

```diff
@@ -1268,8 +1274,8 @@ etcd:
   # -- PodDisruptionBudget settings
   # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
   podDisruptionBudget:
-    enabled: true
-    maxUnavailable: 2
+    enabled: false
+    maxUnavailable: 1
```

Comment:
I think `podDisruptionBudget` should be support both `maxUnavailable` and `minAvailable` options.
in `values.yaml`
```
  podDisruptionBudget:
    enabled: false
    maxUnavailable: 1
    # minAvailable: 1
```
and template files
```yaml
spec:
  {{- with $component.maxUnavailable }}
  maxUnavailable: {{ . }}
  {{- end }}
  {{- with $component.minAvailable }}
  minAvailable: {{ . }}
  {{- end }}
```

Reply:
have done.